src/payment/models/crypto-staking/crypto-staking.service.ts

```diff
@@ -70,6 +70,7 @@ export class CryptoStakingService {
 
     // update staking balance
     await this.stakingService.updateBalance(entity.stakingRoute.id);
+    await this.updateStakingBalance([entity.stakingRoute?.id]);
```

Comment:
Ich würde diese beiden Methoden zusammenfassen. Also die Logik von updateStakingBalance in updateBalance verschieben.